packages/next/client/head-manager.ts

```diff
@@ -60,7 +60,7 @@ function updateElements(type: string, components: JSX.Element[]): void {
   for (
     let i = 0, j = headCountEl.previousElementSibling;
     i < headCount;
-    i++, j = j!.previousElementSibling
+    i++, j = j?.previousElementSibling || null
```

Comment:
```suggestion
    i++, j = j?.previousElementSibling
```

I don't think null is necessary

Reply:
The second commit that related to this. 

"Type 'undefined' is not assignable to type 'Element | null'"

Thanks.